proposals/description/tip_118.ts

```diff
@@ -0,0 +1,542 @@
+          addresses.daiHoldingPCVDeposit,
+          addresses.voltHoldingPCVDeposit,
+          addresses.gOHMHoldingPCVDeposit
+        ]
+      ],
+      description: 'Set all new ERC20 holding deposits as safe addresses on the PCV Guardian'
+    },
+
+    ///  PCV DRIP CONTROLLERS
+    {
+      target: 'aaveEthPCVDripController',
+      values: '0',
+      method: 'pause()',
+      arguments: (addresses) => [],
+      description: 'Pause the ETH PCV drip controller'
+    },
+    {
+      target: 'raiPCVDripController',
+      values: '0',
+      method: 'pause()',
+      arguments: (addresses) => [],
+      description: 'Pause the RAI PCV drip controller'
+    },
+    {
+      target: 'lusdPCVDripController',
+      values: '0',
+      method: 'pause()',
+      arguments: (addresses) => [],
+      description: 'Pause the LUSD PCV drip controller'
+    },
+
+    ///// Deprecate PSM skimmers
+    {
+      target: 'lusdPSMFeiSkimmer',
+      values: '0',
+      method: 'pause()',
+      arguments: (addresses) => [],
+      description: 'Pause the LUSD PSM Fei skimmer'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeRole(bytes32,address)',
+      arguments: (addresses) => [ethers.utils.id('PCV_CONTROLLER_ROLE'), addresses.lusdPSMFeiSkimmer],
+      description: 'Revoke PCV_CONTROLLER_ROLE from lusdPSMFeiSkimmer'
+    },
+
+    {
+      target: 'ethPSMFeiSkimmer',
+      values: '0',
+      method: 'pause()',
+      arguments: (addresses) => [],
+      description: 'Pause the ETH PSM Fei skimmer'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeRole(bytes32,address)',
+      arguments: (addresses) => [ethers.utils.id('PCV_CONTROLLER_ROLE'), addresses.ethPSMFeiSkimmer],
+      description: 'Revoke PCV_CONTROLLER_ROLE from ethPSMFeiSkimmer'
+    },
+
+    //// Activate DAI PSM skimmer and burn Fei
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantRole(bytes32,address)',
+      arguments: (addresses) => [ethers.utils.id('PCV_CONTROLLER_ROLE'), addresses.daiFixedPricePSMFeiSkimmer],
+      description: 'Grant PCV_CONTROLLER_ROLE to daiFixedPricePSMFeiSkimmer'
+    },
+    {
+      target: 'daiFixedPricePSMFeiSkimmer',
+      values: '0',
+      method: 'skim()',
+      arguments: (addresses) => [],
+      description: 'Burn excess Fei on the DAI PSM. Will burn ~135M FEI'
+    },
+
+    ///// Send VOLT to it's holding ERC20 deposit
+    {
+      target: 'volt',
+      values: '0',
+      method: 'transfer(address,uint256)',
+      arguments: (addresses) => [addresses.voltHoldingPCVDeposit, '10000000000000000000000000'],
+      description: 'Move 10M VOLT from the DAO timelock to the VOLT holding PCV deposit'
+    },
+
+    ///// Send gOHM to it's holding ERC20 deposit
+    {
+      target: 'gOHM',
+      values: '0',
+      method: 'transfer(address,uint256)',
+      arguments: (addresses) => [addresses.gOHMHoldingPCVDeposit, '577180000000000000000'],
```

Comment:
Would prefer these numbers be in a readable format, ie ethers.utils.WeiPerEther.mul(5_771_800)... etc

Reply:
For all the transfers that involve specific amounts being transferred, I read the exact balance of the contract off Etherscan and have passed that returned string in here. Feel it removes any chance of a mistake, so will keep as is for these specific values. 

Definitely agree with putting the numbers in readable format generally